src/main/java/auto_ckz/common/enums/RepairStatusWrapper.java

```diff
@@ -0,0 +1,20 @@
+package auto_ckz.common.enums;
+
+
+import java.util.ArrayList;
+
+public class RepairStatusWrapper {
```

Comment:
Próbowałem wykorzystać Twoje rozwiązanie, oczywiście w Twoim przypadku się sprawdza, jeśli chcesz tylko wylistować, ale u mnie akurat, jak zamierzasz użyć tego w formularzu w select'e to juz się nie sprawdzi